### DISCUSSION TITLE
Auto-Generated (Created On Date) field times are innacurate

### DISCUSSION THREAD

**Discussion by jacobmaussocc** [Budibase/budibase#11729] [Help] [answered]
All of the timestamps for my created on field is getting set five hours back and since this value is automatically created with the entries I am not sure why this is, or how to adjust it.

**Comment by ConorWebb96** [comment]
Hey @jacobmaussocc,

- What version of Budibase are you using?
- Are you self-host or cloud?
- What data source are you using?
- What timezone are you in?
- What browser are you using?

**Reply by jacobmaussocc** [reply]
2.9.3
Self-host
MySQL data source
CST
Chrome

**Reply by ConorWebb96** [reply]
Hey @jacobmaussocc,

What timezone is your database set to? You should be able to run the query `show timezone` in your database to find this out.
When you do can you let me know what the output is. 

Thanks!

**Reply by jacobmaussocc** [reply]
Database is CDT time, so the same. I know the times in the database are accurate.

**Reply by ConorWebb96** [reply]
Hey @jacobmaussocc,

Are you using a docker database or is it hosted elsewhere?

**Reply by jacobmaussocc** [reply]
Docker, I have help request you were helping me out with multiple row deletion not working, this is the same instance of budibase.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @jacobmaussocc,

Ahh cheers thanks for reminding me. 

I'm going to assume that your tz(time zone) on your docker environment is UTC as this is what its set to by default. If you haven't changed it when setting up your self-host environment then this is definitely the case.

Could you change your timezone to CDT. This should fix your auto generated dates displaying incorrectly. To do this you will need to go into your `docker-compose.yaml` file and add one of these to your environments within. `TZ: America/Chicago` or - `TZ=America/Chicago`. I've supplied some screenshots below to show were these should go within the yaml file.

<img width="470" alt="Screenshot 2023-09-14 at 05 10 10" src="https://github.com/Budibase/budibase/assets/126772285/ce55e061-1272-4a62-af68-0576d6f89018">

<img width="299" alt="Screenshot 2023-09-14 at 05 13 15" src="https://github.com/Budibase/budibase/assets/126772285/ab56cdbf-f591-4361-aabf-978bbffd2e8b">

<img width="212" alt="Screenshot 2023-09-14 at 05 13 52" src="https://github.com/Budibase/budibase/assets/126772285/69d930fc-4f05-4773-86ff-098122f175eb">

Continue this for the majority of the environments and it should fix your time zoning issues. 

Make sure to **restart** your **docker containers** after making this change otherwise it won't do anything.

I hope this helps.

**Reply by jacobmaussocc** [reply]
Thanks champ.
